locales/de/tools.ftl

```diff
@@ -2,12 +2,12 @@
 
 tools-page-title = Werkzeuge
 tools-editor-support-heading = Erstklassige Editor-Unterstützung
-tools-editor-support-description = Egal ob du gerne auf der Kommandozeile arbeitest, oder lieber einen mächtigen grafischen Editor verwendest, es gibt eine Integration für deinen Lieblingseditor. Oder du kannst deine Eigene bauen mithilfe des <a href="https://github.com/rust-lang/rls">Rust Language Servers</a>.
+tools-editor-support-description = Egal ob du gerne auf der Kommandozeile arbeitest, oder lieber einen mächtigen grafischen Editor verwendest, es gibt eine Integration für deinen Lieblingseditor. Oder du kannst deine Eigene bauen mithilfe des <a href="https://github.com/rust-lang/rls">Rust Language Server</a>.
```

Comment:
I assume this 's' was unintentional, for one since it wasn't in any of the other locales, so I removed it.